Resources/Prototypes/Guidebook/chemicals.yml

```diff
@@ -0,0 +1,60 @@
+- type: guideEntry
+  id: Chemicals
+  name: guide-entry-chemicals
+  text: "/ServerInfo/Guidebook/Chemicals.xml"
+  children:
+  - Elements
+  - Medicine
+  - Narcotics
+  - Pyrotechnic
+  - Toxins
+  - Foods
+  - Botanical
+  - Biological
+  - Others
+  filterEnabled: True
+
+- type: guideEntry
+  id: Elements
+  name: guide-entry-elements
+  text: "/ServerInfo/Guidebook/Chemical Tabs/Elements.xml"
+
+- type: guideEntry
+  id: Medicinal
```

Comment:
Please do not use a subtle naming difference like "Medicine" vs "Medicinal" to distinguish this page with the one from teh jobs tab.